lib/provider/soundcloud.js

```diff
@@ -22,7 +22,7 @@ module.exports = SoundCloud;
 
 SoundCloud.prototype.parseUrl = function(url, result) {
   var match = url.match(
-    /soundcloud\.com\/(?:([\w-]+)\/(sets\/)?)([\w-]+)/i
+    /(?:m\.)?soundcloud\.com\/(?:([\w-]+)\/(sets\/)?)([\w-]+)/i
```

Comment:
Should this also support creating mobile urls then?

Reply:
I don't think there is any benifit to it. The `m.soundcloud.com` url is automatically expanded (at least by my observations) so I don't think there is any use for that.